src/core/ext/filters/client_channel/subchannel_stream_client.cc

```diff
@@ -369,6 +369,10 @@ void SubchannelStreamClient::CallState::RecvInitialMetadataReady(
 }
 
 void SubchannelStreamClient::CallState::RecvMessageReady() {
+  if (!recv_message_.has_value()) {
+    call_->Unref(DEBUG_LOCATION, "recv_message_ready");
+    return;
+  }
   // Concatenate the slices to form a single string.
```

Comment:
This code can probably now be replaced with just `std::string recv_message = recv_message_->JoinIntoString()`.